added comment

lhubbard-USGS opened the pull request:
hooray

Commit:
added comment

lhubbard-USGS committed:
added comment

lhubbard-USGS committed:
Merge branch 'master' of github.com:lhubbard-USGS/trainR

scworland commented on R/calc_water_year.R:
```diff
@@ -16,4 +16,4 @@ calc_water_year <- function(date.vec){
   return(water.year)
 }
 
-#testing: conflict resolution
```
testing this feature

rbcarvin commented on R/calc_water_year.R:
```diff
@@ -16,4 +16,4 @@ calc_water_year <- function(date.vec){
 
-#testing: conflict resolution
\ No newline at end of file
+################
```
:zap: okey dokey

scworland replied:
still testing

scworland reviewed (approved):
test

scworland reviewed (approved):
test

lhubbard-USGS committed:
merge from upstream and fixed conflict

Merge branch 'master' of github.com:USGS-R/trainR

# Conflicts:
#	R/calc_water_year.R
